src/schema/top_level_files.yaml

```diff
@@ -0,0 +1,41 @@
+README:
+  required: True
+  extensions:
+    - .md
+    - .rst
+    - None
+CHANGES:
+  required: True
+  extensions:
+    - .md
+    - .rst
+    - None
+dataset_description:
+  required: True
+  extensions:
+    - .json
+genetic_info:
+  required: False
+  extensions:
+    - .json
+participants:
+  required: False
+  extensions:
+    - .tsv
+    - .json
+phasediff:
+  required: False
+  extensions:
+    - .json
+phase1:
+  required: False
+  extensions:
+    - .json
+phase2:
+  required: False
+  extensions:
+    - .json
+fieldmap:
+  required: False
+  extensions:
+    - .json
```

Comment:
Unclear why fieldmap JSONs are being explicitly listed at the top level. Any sidecar that applies to all participants can be specified at this level.

Reply:
This was something I adapted from the [top_level_rules in bids-validator](https://github.com/bids-standard/bids-validator/blob/029e37e2d77702a9c130a365b5480c3a41461297/bids-validator/bids_validator/rules/top_level_rules.json#L13-L16). I wasn't sure why they were included there but decided to err on the side of caution.

I can remove them.

Reply:
Removed in bfac102.